### PR TITLE
SS-28 Added finding rooms by dormitoryId

### PR DESCRIPTION
**Pull request by Yevhensh**

### CODE REVIEW

File: src/main/java/edu/students/settler/model/entity/Dormitory.java [hunk 1]
```diff
@@ -19,8 +20,9 @@
     @Column(name = "number")
     private int number;
 
-    @OneToMany(fetch = FetchType.EAGER, cascade = CascadeType.ALL)
+    @OneToMany(fetch = FetchType.LAZY, cascade = CascadeType.ALL)
     @JoinColumn(name = "dormitory_id")
+    @JsonIgnore
```
Comment (MoGUREV) [inline src/main/java/edu/students/settler/model/entity/Dormitory.java]: What this mean?

File: src/main/java/edu/students/settler/service/impl/DormitoryServiceImpl.java [hunk 2]
```diff
@@ -19,4 +22,9 @@
     public List<Dormitory> findAll() {
         return dormitoryRepository.findAll();
     }
+
+    @Override
+    public Set<Room> findDormitoryRooms(long dormitoryId) {
+        return dormitoryRepository.findById(dormitoryId).map(Dormitory::getRooms).orElse(Collections.emptySet());
```
Comment (MoGUREV) [inline src/main/java/edu/students/settler/service/impl/DormitoryServiceImpl.java]: Not critical.

I suggest formatting code when you using `Optional`:
```java
return dormitoryRepository.findById(dormitoryId)
               .map(Dormitory::getRooms)
               .orElse(emptySet());
```
Also, use `static import` for `Collections` methods.

Comment (Yevhensh) [reply src/main/java/edu/students/settler/service/impl/DormitoryServiceImpl.java]: matter of taste :)

Comment (Yevhensh) [reply src/main/java/edu/students/settler/service/impl/DormitoryServiceImpl.java]: i understand that style i also was writing in that style when coding on java, but after i rethinked this style and now doing it oneline :)

**Pull request merged**